CHANGELOG.rst

```diff
@@ -12,10 +12,14 @@ Backward-incompatible changes:
 
 - Dropped support for Python 3.6.
 - The minimum ``cryptography`` version is now 41.0.0.
+- Removed ``OpenSSL.crypto.loads_pkcs7`` and ``OpenSSL.crypto.loads_pkcs12`` which had been deprecated for 3 years.
```

Comment:
Guess it should be `OpenSSL.crypto.load_pkcs7_data` and `OpenSSL.crypto.load_pkcs12`?

#1251 changed L20 from `OpenSSL.crypto.loads_pkcs12` to `OpenSSL.crypto.load_pkcs12`.